plugin/trino-hudi/pom.xml

```diff
@@ -110,6 +115,12 @@
             <artifactId>guice</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Why do we need to add this?

Reply:
maven was complaining that it was missing

Reply:
I found this is due to the accidental usage of `org.apache.commons.lang3.tuple.ImmutablePair` from this dependency. You can replace it with `Map.entry()`.